opta/cli.py

```diff
@@ -63,22 +64,20 @@ def cli() -> None:
         one_time()
         cleanup_files()
         cli()
-    except CalledProcessError as e:
-        logger.exception(e)
-        if e.stderr is not None:
-            logger.error(e.stderr.decode("utf-8"))
-        sys.exit(1)
     except UserErrors as e:
-        logger.error(e)
+        logger.error(str(e))
         logger.info(
-            "If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev"
+            f"{fg('magenta')}If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev{attr(0)}"
         )
         sys.exit(1)
     except Exception as e:
-        logger.exception(e)
+        logger.exception(str(e))
         logger.info(
-            "If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev"
+            f"{fg('red')}Unhandled error encountered -- a crash report zipfile has been createded for you. "
```

Comment:
Why a zipfile and not a single text file?

Reply:
It makes things so much easier for us (debugging)

Reply:
if it was difficult, I would have just made a single file but it was so simple to do the zip and so nice to see the contents all neatly displayed

Reply:
if you disagreee I could certainly make it a single file